### PR TITLE
file2.txt change from test_br2

### PR DESCRIPTION
**Pull request by gerykahn**
Please review

### CODE REVIEW

File: file2.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 file 2
 file 2
```
Comment (dr942120) [inline file2.txt:2]: ochen' interesno

**Comment by gerykahn** [comment]
It is OK

File: file2.txt [hunk 2]
```diff
@@ -1,4 +1,5 @@
...
 file 2
-mew line
\ No newline at end of file
+mew line
+changed from test_br2
```
Comment (dr942120) [inline file2.txt:5]: interesting #2

**Review by dr942120 — COMMENTED** [review]
comment#2

**Review by dr942120 — CHANGES_REQUESTED** [review]
change 1,2,3

**Pull request merged**